src/spicy-mayo.md

```diff
@@ -0,0 +1,27 @@
+# Spicy Mayo
+
+A simple sauce that is at almost every sushi bar in the United States.
+
+- ⏲️ Prep time: 3 min
+
+## Ingredients
+
+- chili sauce (e.g Sriracha)
+- mayonnaise
+- sesame oil
+
+## Directions
+
+1. Add 2 parts mayonnaise to 1 part chili sauce in a bowl.
+2. Add 1 tbsp of sesame oil for every cup of mayonnaise you use.
+3. Mix all of the ingredients together, and taste test it for more chili sauce or mayonnaise.
+
+## Note
+
+This is just a basic guide, adapt this recipe to your liking.
+
+## Contribution
+
+- scary90
+
+;tags: simple sauce japanese
```

Comment:
```suggestion
;tags: basic sauce japanese
```

Just to avoid adding another tag to the long list, I think using `basic` instead of simple would be best. I think this would also fit in with the basic category